packages/server/src/auth/auth.service.ts

```diff
@@ -42,13 +44,15 @@ export class AuthService {
    * @returns JWT or 401 status code
    */
   async validateEmail(projectId: string, email: string, password: string): Promise<any> {
-    if (projectId != null) {
-      const user = await this.userService.findUserByEmail(projectId, email);
+    if (projectId == null || email == null || password == null) {
+      throw new HttpException('Bad request', HttpStatus.BAD_REQUEST);
```

Comment:
Missed this earlier, may just want to tweak the message to include the fact that the `BAD_REQUEST` is due to missing one of the required parameters